resources/views/dashboard.blade.php

```diff
@@ -114,8 +113,9 @@
             connect() {
                 this.pusher = new Pusher(this.app.key, {
                     wsHost: this.app.host === null ? window.location.hostname : this.app.host,
-                    wsPort: this.port,
-                    wssPort: this.port,
+                    wsPort: this.app.port === null ? 6001 : this.app.port,
+                    wssPort: this.app.port === null ? 6001 : this.app.port,
+                    wssPath: this.app.path === null ? '' : this.app.path,
```

Comment:
I believe there must be `wsPath`

Reply:
Correct ill change that later
```suggestion
                    wsPath: this.app.path === null ? '' : this.app.path,
```